### PR TITLE
The Darwin FML backend.

### PR DESCRIPTION
**Pull request by chinmaygarde**

### CODE REVIEW

File: fml/platform/darwin/message_loop_darwin.mm [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/fml/platform/darwin/message_loop_darwin.h"
+
+#include <Foundation/Foundation.h>
+
+namespace fml {
+
+static const CFTimeInterval kDistantFuture = 1.0e10;
```
Comment (abarth) [inline fml/platform/darwin/message_loop_darwin.mm]: contexpr :)

**Comment by abarth** [comment]
LGTM

File: fml/platform/darwin/message_loop_darwin.mm (hunk 1 above)
Comment (chinmaygarde) [inline fml/platform/darwin/message_loop_darwin.mm]: Damn. I caught this in the Android backend but forgot to check here. Fixed.

**Comment by chinmaygarde** [comment]
Licenses fixed.

**Pull request merged**